code/code/misc/combat.cc

```diff
@@ -1333,17 +1333,34 @@ bool TObj::makeScraps()
 
 static void absorb_damage(TBeing *v, wearSlotT part_hit, int *dam)
```

Comment:
I don't see this code modifying `v`, let's constify it.

Reply:
You are right on all your points. I focused on updating the business logic and forgot to ask myself if the code sucks. Would this be better as:
- a member function of Tbeing that accepts part_hit and returns an absorb amount
- an anonymously namespaced function in combat.cc that accepts a const victim and part hit and returns an absorb amount?

Reply:
I'd choose the 2nd.